scripts/deploy.actions.js

```diff
@@ -59,10 +62,19 @@ class DeployActions extends BaseScript {
     const manifestPackage = manifest.packages[this.config.manifest.packagePlaceholder]
     manifestPackage.version = this.config.app.version
     const relDist = this._relApp(this.config.actions.dist)
-    await Promise.all(Object.entries(manifestPackage.actions).map(async ([name, action]) => {
+    Object.entries(manifestPackage.actions).forEach(([name, action]) => {
       // change path to built action
       action.function = path.join(relDist, name + '.zip')
-    }))
+      // attempt to parse config
+      if (action.inputs) {
+        Object.entries(action.inputs).forEach(([key, value]) => {
+          if (value.startsWith(AIO_INPUT_PLACEHOLDER)) {
+            action.inputs[key] = aioConfig.get(value.slice(AIO_INPUT_PLACEHOLDER.length)) || ''
```

Comment:
Where is this expected to be defined?
Can you write a use-case to help me understand this?
Are values coming from .aio / .env ?
When aio-lib-config loads .env it will convert AIO_INPUT_PLACEHOLDER_ into "input": { "placeholder": ... } } so we might be able to leverage that for a clearer api.

Reply:
@purplecabbage thanks for the review, sorry for the late response, missed it somehow.

Here the idea is to provide a way for the user to load an AIO config value into an action parameter, this is especially useful for loading an entire object. This was develop with ims jwt token generation in mind where the action needs the ims jwt config (client secret, id, org, private key, scopes, ...) to generate the token, and this configuration is likely available in AIO config as one object under the `$ims.jwt` key (when using `@adobe/aio-lib-core-ims` or auth plugin).
 
This can be useful for any stored config object in .env and/or/mixed .aio, as the current way is a bit cumbersome: specify each variable individually as env vars and link them in the manifest default inputs. 

AIO_INPUT_PLACEHOLDER is just a constant name in `scripts/deploy.actions.js` and holds the string `$aio.`. This is a prefix that indicates to app-scripts deploy to replace `$aio.<user-specified-config-key>` by `aioConfig.get(user-specified-config-key)`.

here is an example:
- user's `.aio`:
```
{
  some_config: {
    key1: 'xyz',
    key2: 'abc'
  }
}
```
- user's `.env`:
```
AIO_SOME__CONFIG_KEY3=secret
```
- user's `manifest.yml` (partial)
```
      action:
        function: actions/action.js
        web: yes
        runtime: 'nodejs:10'
        inputs:
          config: $aio.some_config
```

- user's `actions/action.js` (partial):

```
function main ({config, ...params}) {
  // config holds key1, key2, key3
}
```

Reply:
👍